### PR TITLE
feat: add bulkDealerFeedback

### PR DESCRIPTION
**Pull request by Marine-Berthier**
References [CAR-7202](https://autoricardo.atlassian.net/browse/CAR-7202?atlOrigin=eyJpIjoiOTU4ZmUyNDAxOWQ2NGM4YWJlMDU5NjU0NWU3YjI0YmIiLCJwIjoiaiJ9)
https://car-sale-tracking-service.preprod.carforyou.ch/swagger-ui/index.html#/Dealer%20Feedback/bulkCreateDealerFeedbackUsingPOST

## Motivation and context

Add new api to store bulk data collected by the dealer feedback.

Thank you 😃

### CODE REVIEW

File: src/services/carSaleTracking/bulkDealerFeedback.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    purchaseConfirmed: boolean
+  }[]
+  options?: ApiCallOptions
+}): Promise<void> => {
+  return postData({
+    path: `dealers/${dealerId}/listings/car-sale/dealer-feedback`,
+    body: {
+      elements,
+    },
+    options,
```
Comment (GutierrezGo) [inline src/services/carSaleTracking/bulkDealerFeedback.ts:20]: I think in options you need the authorized Request.

**Review by ufkaya — APPROVED** [review]
LGTM

File: src/services/carSaleTracking/bulkDealerFeedback.ts (hunk 1 above)
Comment (lkappeler) [reply src/services/carSaleTracking/bulkDealerFeedback.ts:20]: IIRC it's only one of the purposes of the options they are also used to set custom fetch options.
Regarding auth I would assume that the request needs authentication as the single feedback one does @Marine-Berthier can you check that with backend and add it if needed

Comment (Marine-Berthier) [reply src/services/carSaleTracking/bulkDealerFeedback.ts:20]: Yes, will do !

**Review by masone — APPROVED** [review]
nice one! high five! ✋

**Pull request merged**